datamodel/datamodel.md

```diff
@@ -0,0 +1,21 @@
+Data Model Explanation
```

Comment:
I don't think we need to document this here and in the README. I don't think we need this file and the `datamodel` folder.